woocommerce/woocommerce-android
# [Scan to update inventory] Part 4 – Add View Product Details button

Closes: #10284 


### Description

This PR adds the View Product Details button to the Scan to update nventory bottomsheet. 

### Testing instructions


Go to the products tab
click the scan Icon
Scan barcode with the SKU of existing product (you can use online barcode generator like [this one](https://barcode.tec-it.com/en/?data=ACGADO))
Once the product is displayed, make sure the View Product Details button is displayed 
click on the button and make sure it takes you to the correct product details (test on a variation as well)

### Images/gif


<img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/c3bde34a-5618-478b-aecb-179000aac4f2" width=250>
